core/server/src/main/java/alluxio/worker/block/ClientRWLock.java

```diff
@@ -46,6 +46,13 @@ public Lock writeLock() {
   }
 
   /**
+   * @return the reference count
+   */
+  public Integer getReferenceCount() {
```

Comment:
actually, if `validate()` is moved to test, then this method probably also belongs to test